package.json

```diff
@@ -42,6 +42,7 @@
     "gulp-insert": "0.5.0",
     "gulp-less": "3.1.0",
     "gulp-load-plugins": "1.3.0",
+    "gulp-mocha": "^3.0.1",
```

Comment:
Will do. Consider [`npm shrinkwrap`](https://docs.npmjs.com/cli/shrinkwrap) if the goal is freezing the dependency tree. Using exact numbers brings a maintenance cost and still does not lock down 2nd level dependencies.